.github/workflows/ceramic-promote-production.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy Ceramic Node to Production
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: "Commit ref"
+        required: true
+        type: string
+
+jobs:
+  deploy-production:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          cache: "yarn"
+          cache-dependency-path: infra/ceramic-mainnet/package-lock.json
+      # Update the pulumi stack with new image
+      - run: |
+          npm install
+          pulumi stack select -c gitcoin/ceramic/mainnet-node
+          pulumi config -s gitcoin/ceramic/mainnet-node set aws:region us-east-1 --non-interactive
+        working-directory: infra/ceramic-mainnet
+        env:
```

Comment:
I believe there are some additional values we need to pass in.

I know I include `ROUTE_53_ZONE` and `DOMAIN`

Reply:
Those are passed in in the step below, as env vars